src/m365/spo/commands/list/ListInstance.ts

```diff
@@ -1,4 +1,5 @@
 export interface ListInstance {
+  RoleAssignments: RoleAssignment[];
```

Comment:
Let's sort properties alphabetically so that it's easier to look them up in the future.